tools/checkpoint_conversion/convert_basnet_checkpoints.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+"""Converts BASNet model weights to KerasHub.
+
+Usage: python3 convert_basnet_checkpoint.py
+
+Downloads BASNet modelweights with ResNet34 backbone and converts them to a
+KerasHub model. Credits for model training go to Hamid Ali
+(https://github.com/hamidriasat/BASNet).
+
+Requirements:
+pip3 install -q git+https://github.com/keras-team/keras-hub.git
+pip3 install -q gdown
+"""
+
+import gdown
```

Comment:
let's avoid a weird deps like this. We don't use it elsewhere. Should be able to make a bare link to a google drive for now, but more importantly this isn't converting anything, it's just wrapping some weights into a preset.